client/landing/stepper/declarative-flow/internals/steps-repository/sensei-plan/create-sensei-site.ts

```diff
@@ -54,20 +54,20 @@ export const useCreateSenseiSite = () => {
 		[]
 	);
 
-	const [ progress, setProgress ] = useState< Progress >( {
-		percentage: 0,
-		title: '',
-	} );
-
 	const { __ } = useI18n();
 	const locale = useLocale();
 	const visibility = useNewSiteVisibility();
 
-	const createAndConfigureSite = useCallback( async () => {
```

Comment:
The changes on this file are just to make sure we have a default progress title.